### ISSUE TITLE
Pull fonts and colors from the DH design system

### ISSUE THREAD

**Issue by TravisWyche** [HausDAO/daohaus-monorepo#467]
Need a few type faces and colors for the Docusaurus

**Comment by jonathanprozzi** [comment]
@earth2travis Our custom font works for this but having more trouble with the colors. This is due to the nextra theme being a 'plugin' basically and it's using Tailwind under the hood. I've tried a few things but nothing has been consistent. Going to try a few more things but then I'll at least push up a PR using Mulish to match the apps.

We can then keep figuring out the custom theme.

**Comment by jonathanprozzi** [comment]
Ok, I got this working kind of. I successfully updated to Nextra v2 (beta, but has nice features and will be stable soon!) and took care of the breaking changes. In that we're able to set a `primaryHue`  -- this isn't as deeply customizable as a full theme config, but we can set a main accent color based on hues. 

I'm going to get this stable and make a PR for the v2 migration and then I'll update the fonts and styles.

**Comment by earth2travis** [comment]
@jonathanprozzi sounds good. Just make sure it deals with how we have the menus collapsed with Plor's fix and if the navigation structure changes any that all the links still work sitewide

**Comment by jonathanprozzi** [comment]
> @jonathanprozzi sounds good. Just make sure it deals with how we have the menus collapsed with Plor's fix and if the navigation structure changes any that all the links still work sitewide

Yup should be good. It's a setting now. Will be sure to test that tho
